### DISCUSSION TITLE
LF APAC开源布道者分享系列之五：布道者的修行 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#8] [Announcements]
# LF APAC开源布道者分享系列之五：布道者的修行 - 开源之道

https://opensourceway.community/posts/lfapac-evangelist-sharing/advocate-how-to-learning-by-yourself/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

其实, Evangelist 并不是一个具体职位, 而应该是一种生源风格;
就象当年主动去世界边缘几年/十几年/几十年/...将一生投入到传教的新教教士们一样;

他们并没有工资, 只有不固定的补给,
特别是来中国的那些, 除了自身带来的, 就只有真诚和学识了...
他们建立的影响力, 完全是通过学习达成的:

- 学习当地语言
- 学习当地文化
- 学习当地生活
- 学习...一切在地的所有

并积极融入其中, 只是在必要时, 才给出相同问题自己的见解,
当然, 以及遵守教义约定的所有...

以自身长期真实行为来给当地人民建立一系列全新概念,
将自己作为形象来塑造...

最后人戏合一, 自己都分不清, 自己原先的生活应该是如何的了...

开源Evangelist 也一样, 并不是说演讲才是布道,
自己日常生活本身才是更加强大的布道...

所以, 如果你用一台 WIndows10 系统电脑, 打开 PPT 来演说 开源力量,
那么就别怪程序猿们看不起你了吧...
